src/x_net.c

```diff
@@ -248,6 +249,10 @@ static void netsend_connect(t_netsend *x, t_symbol *s, int argc, t_atom *argv)
     }
     memcpy((char *)&x->x_server.sin_addr, (char *)hp->h_addr, hp->h_length);
 
+    /* multicast? */
+    if (0xE0000000 == (ntohl(x->x_server.sin_addr.s_addr) & 0xF0000000))
```

Comment:
multicast should only be enabled for UDP (this is minor, as the check only effects the debugging printout, but stil...)

Reply:
Also fixed.